### DISCUSSION TITLE
How to connect Mixtral 8x7b instructor in windows application using C++?

### DISCUSSION THREAD

**Discussion by aitechguy0105** [ggml-org/llama.cpp#5883] [General]
I want to use Mixtral8x7b instrutor in windows application.
Is it possible to connect using C++?

**Comment by jetro30087** [comment]
Yes, llama.cpp can do that.

**Reply by aitechguy0105** [reply]
Thanks. I have implemented that. Then is it possible to connect with T5 model?
I would really appreciate it if you could provide example.

**Reply by jetro30087** [reply]
I've never tried connecting two models. But you probably could just load two into memory if you have enough ram. If that doesn't work, then loading/unloading the appropriate model in your program as needed would work just fine. I don't really have an example to provide, but the repo provides example programs for every feature they have in the examples folder.

**Reply by aitechguy0105** [reply]
Thanks for your kind help.
I will study examples.
Thanks.
